web/templates/settings_overlay.hbs

```diff
@@ -191,7 +191,7 @@
             </div>
         </div>
         <div id="settings_content" data-simplebar data-simplebar-auto-hide="false">
-            <div class="organization-box organization">
+            <div class="organization-box organization" data-simplebar>
```

Comment:
This is not the correct fix for this I guess. I can see the native scrollbar sometimes. Also we already have `data-simplebar` element for `#settings_content` element. I think we should remove `overflow-x: hidden` set for `#settings_content` element and instead set it to `auto`.